src/plugins/saved_search/public/services/saved_searches/saved_search_attribute_service.ts

```diff
@@ -98,6 +98,7 @@ export const toSavedSearch = async (
       ...splitReferences(result.attributes),
       savedSearchId: id,
       sharingSavedObjectProps,
+      managed: false,
```

Comment:
This is used for loading saved search panels on dashboards. Does it matter that they'll always be marked as unmanaged?

Reply:
It actually doesn't matter, but I will update it for consistency's sake 👍

Reply:
Done in https://github.com/elastic/kibana/pull/175256/commits/bac82b20e371fd4693c884075643394eaf7d9cc7